src/server/server.ts

```diff
@@ -19,8 +19,17 @@ process.on('unhandledRejection', (e) => {
   process.exit(1);
 });
 
+process.on('SIGINT', () => {
+  console.log('SIGINT received. Exiting...');
+  process.exit(0);
+});
+
 const router = express();
-router.use('/graphql', graphqlHTTP({
+
+// TODO: Replace this with a reverse proxy
+router.use(express.static('dist/public'))
```

Comment:
Using reverse proxy now makes the docker file and compose looks very complicated. Let's do this for now and deal with this later if we have an actual perf problem.